src/components/GeneralTableRow.tsx

```diff
@@ -5,30 +5,39 @@ import {grey} from "../themes/colors/aptosColorPalette";
 interface TableRowProps {
   children?: React.ReactNode;
   onClick: () => void;
+  clickDisabled?: boolean;
 }
 
-export default function GeneralTableRow({children, onClick}: TableRowProps) {
+export default function GeneralTableRow({
+  children,
+  onClick,
+  clickDisabled,
+}: TableRowProps) {
   const theme = useTheme();
   return (
     <TableRow
       onClick={onClick}
       sx={{
-        cursor: "pointer",
+        cursor: clickDisabled ? undefined : "pointer",
```

Comment:
Does `clickDisabled` actually prevent clicking or does it just adjust the visual style? Might want to adjust the onClick callback to exit early if clickDisabled is true.

Reply:
You're right, `onClick` would still work. Thanks for catching it. I love all your comments!!